imports/api/entries/entries.js

```diff
@@ -15,6 +15,20 @@ let Schema = new SimpleSchema({
   lookingFor: String,
   oneLineIntro: String,
   verified: Boolean,
+  flags: {
+    type: Array,
+    optional: true,
+  },
+  'flags.$': {
```

Comment:
Out of curiosity, where is $ defined or what does it typically mean in meteor/mongo?

Reply:
The $ there reflects the value of the `flags` property. Here's the doc:

![image](https://user-images.githubusercontent.com/4512699/56866869-5cd3bc80-6993-11e9-83a5-98057dee9817.png)

https://github.com/aldeed/simple-schema-js